api-interop-layer/data/db.js

```diff
@@ -26,22 +26,38 @@ const getDatabaseConnection = () => {
     port: process.env.DB_PORT ?? 3306,
     ssl: { rejectUnauthorized: false },
   };
-}
+};
+
+let pool = false;
```

Comment:
`null`? `false` kinda threw me for a loop here

Reply:
This is a personal preference to indicate that the pool hasn't been created yet, but I can totally see why that's not intuitive. I'll change it so that it's not defined at all, just declared. It'll have the same effect but be less confusing.